README.md

```diff
@@ -1,93 +1,64 @@
-<p align="center"><img src="https://firefly-iii.org/static/img/logo-small-new.png"></p>
+# Firefly III for YunoHost
 
-# Firefly III v4.7.9 for YunoHost
+[![Integration level](https://dash.yunohost.org/integration/firefly-iii.svg)](https://dash.yunohost.org/appci/app/firefly-iii) ![](https://ci-apps.yunohost.org/ci/badges/firefly-iii.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/firefly-iii.maintain.svg)  
+[![Install Firefly III with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=firefly-iii)
 
-[![Integration level](https://dash.yunohost.org/integration/Firefly-III.svg)](https://ci-apps.yunohost.org/jenkins/job/firefly-iii%20%28Community%29/lastBuild/consoleFull)  
+> *This package allows you to install Firefly III quickly and simply on a YunoHost server.  
+If you don't have YunoHost, please consult [the guide](https://yunohost.org/#/install) to learn how to install it.*
 
-[![Install Firefly III with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=firefly-iii)<br><br>
-
-
-*Please note that this app will install PHP 7.2*
-
-## Introduction
+## Overview
 "Firefly III" is a (self-hosted) manager for your personal finances. It can help you keep track of your expenses and income, so you can spend less and save more. Firefly III supports the use of budgets, categories and tags. It can import data from external sources and it has many neat financial reports available. You can [read all about it in the main repository](https://github.com/firefly-iii/firefly-iii) and in the [official documentation](https://firefly-iii.readthedocs.io/en/latest/).
 
-### Interesting links
-
-- [YunoHost project](https://yunohost.org)
-- [Firefly III website](https://firefly-iii.org/)
-- [Firefly III on GitHub](https://github.com/firefly-iii/firefly-iii)
-- [Demo 1](https://demo.firefly-iii.org/login), [Demo 2](http://www.softaculous.com/softaculous/demos/Firefly_III)
+**Shipped version:** 5.2.0
 
-### Screenshots
+## Screenshots
 
-[![The index of Firefly III](https://firefly-iii.org/static/screenshots/4.7.4/tiny/index.png)](https://firefly-iii.org/static/screenshots/4.7.4/index.png) [![The account overview of Firefly III](https://firefly-iii.org/static/screenshots/4.7.4/tiny/account.png)](https://firefly-iii.org/static/screenshots/4.7.4/account.png)
+![](https://camo.githubusercontent.com/e5e5e8e25e7039d15dd75dd132e231263d0b4fb2/68747470733a2f2f7777772e66697265666c792d6969692e6f72672f7374617469632f73637265656e73686f74732f6970686f6e652d636f6d706c6574652e706e67)
```

Comment:
Please don't link to `camo` files directly. It's a cache and the file names change all the time. If you need updated screenshot links, grab them from the original readme of Firefly III.